opm/simulators/linalg/ISTLSolverEbosFlexible.hpp

```diff
@@ -128,7 +147,43 @@ class ISTLSolverEbosFlexible
             assert(recreate_solver == false);
             // Never recreate solver.
         }
-
+	VectorType weights;
+        if( prm_.get<std::string>("preconditioner.type") == "cpr" ||
+	    prm_.get<std::string>("preconditioner.type") == "cprt"
+	    )
+	{	    
+	    bool transpose = false;
+	    if(prm_.get<std::string>("preconditioner.type") == "cprt"){
+		transpose = true;
+	    }
+	    if(prm_.get<std::string>("preconditioner.weight_type") == "quasiimpes") {
+		if(prm_.get<int>("verbosity") > 10){
+		    std::cout << "Using quasiimpes" << std::endl;
```

Comment:
Left over debugging? Otherwise maybe use or logging?

Reply:
Correct logging is the way to go.